src/components/layout/DocsLayout.js

```diff
@@ -278,7 +278,7 @@ function DocsLayout({ children, isLatest: isLatestProp, pageContext }) {
         <Content>
           {tocSectionTitles && (
             <span hidden id="toc-section-titles">
-              {tocSectionTitles}
+              {`Docs > ${tocSectionTitles}`}
```

Comment:
It looks like DocSearch uses "»" as a level seperator:

![image](https://user-images.githubusercontent.com/486540/211634083-59314960-8f74-4bca-90c1-aa447afd4cd9.png)

So, we can match that:

```suggestion
              {`Docs » ${tocSectionTitles}`}
```